libs/@blockprotocol/graph/src/internal/mutate-subgraph.ts

```diff
@@ -11,18 +13,20 @@ import {
 import { isEqual } from "./mutate-subgraph/is-equal.js";
 
 /**
- * Looking to build a subgraph? You probably want `import { buildSubgraph } from @blockprotocol/graph/stdlib`
+ * Looking to build a subgraph? You probably want {@link buildSubgraph} from `@blockprotocol/graph/stdlib`
```

Comment:
As you say, it wasn't written correctly (so not a direct copy paste), and I think the vast majority of people do not manually write their own imports in the world of intellisense and auto-complete. 

I do think this is an improvement for maintainability of the library as if you're clicking through documentation, you can immediately click on `buildSubgraph` and read its documentation with a single click to see if that is actually what you were trying to do. As opposed to the alternative where you have to leave whatever you were using to read this documentation in the first place, write some code to import the other function, and then read its documentation (or you have to use some IDE feature to go to the function rather than importing it).

We really are talking about some miniscule DX improvements here, but I think trying to get our docs more compliant with discoverability in aggregate will do wonders, especially if we struggle to minimise the exposure of various complex parts of the Graph service like temporal versioning.

Reply:
Sure, makes sense – leave it as it is